### PR TITLE
CHE-2521: improve clear recent list action

### PR DESCRIPTION
**Pull request by olexii4**
### What does this PR do?

Move ClearRecentListAction from RecentFileStore. It gives us possibility show action later than before.
### What issues does this PR fix or reference?

Fix visibility of 'Clear list' menu item.
### What issues does this PR fix or reference?
#2521

Signed-off-by: Oleksii Orel oorel@codenvy.com

### CODE REVIEW

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/part/editor/recent/RecentFileStore.java [hunk 1]
```diff
@@ -55,38 +46,29 @@
     private final OpenRecentFilesPresenter openRecentFilesPresenter;
     private final ActionManager            actionManager;
     private final RecentFileActionFactory  recentFileActionFactory;
-    private final CoreLocalizationConstant locale;
-    private final DefaultActionGroup       recentGroup;
 
+    private DefaultActionGroup recentGroup;
```
Comment (ashumilova) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/part/editor/recent/RecentFileStore.java]: formatting

**Review by ashumilova — APPROVED** [review]

File: dashboard/src/app/workspaces/workspace-details/export-workspace/dialog/export-workspace-dialog.controller.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ export class ExportWorkspaceDialogController {
       lineWrapping : true,
       lineNumbers: false,
       matchBrackets: true,
-      readOnly: 'nocursor',
+      readOnly: '',
```
Comment (ashumilova) [inline dashboard/src/app/workspaces/workspace-details/export-workspace/dialog/export-workspace-dialog.controller.js]: readOnly: true - may work as well

**Comment by vzhukovs** [comment]
Ok

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/part/editor/recent/RecentFileStore.java (hunk 1 above)
Comment (olexii4) [reply ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/part/editor/recent/RecentFileStore.java]: done

File: dashboard/src/app/workspaces/workspace-details/export-workspace/dialog/export-workspace-dialog.controller.js (hunk 2 above)
Comment (olexii4) [reply dashboard/src/app/workspaces/workspace-details/export-workspace/dialog/export-workspace-dialog.controller.js]: done

**Review by benoitf — APPROVED** [review]

**Review by vparfonov — APPROVED** [review]

**Pull request merged**